_dependencies/tasks/main.yml

```diff
@@ -4,14 +4,14 @@
   include_vars: { dir: "{{ playbook_dir }}/group_vars/{{ clusterid }}" }
   tags: clusterverse_clean,clusterverse_create,clusterverse_dynamic_inventory,clusterverse_config,clusterverse_readiness
 
-- name: Preflight check
-  block:
-    - assert: { that: "ansible_version.full is version_compare('2.9', '>=')", msg: "Ansible >=2.9 required." }
-    - assert: { that: "app_name is defined and app_name != ''", msg: "Please define app_name" }
-    - assert: { that: "app_class is defined and app_class != ''", msg: "Please define app_class" }
-    - assert: { that: "clusterid is defined and cluster_vars is defined", msg: "Please define clusterid" }
-    - assert: { that: "buildenv is defined and cluster_vars[buildenv] is defined", msg: "Please define buildenv" }
+# - name: Preflight check
+#   block:
+#     - assert: { that: "ansible_version.full is version_compare('2.9', '>=')", msg: "Ansible >=2.9 required." }
+#     - assert: { that: "app_name is defined and app_name != ''", msg: "Please define app_name" }
+#     - assert: { that: "app_class is defined and app_class != ''", msg: "Please define app_class" }
+#     - assert: { that: "clusterid is defined and cluster_vars is defined", msg: "Please define clusterid" }
+#     - assert: { that: "buildenv is defined and cluster_vars[buildenv] is defined", msg: "Please define buildenv" }
 
-    - assert: { that: "(cluster_vars.assign_public_ip == 'yes' and cluster_vars.inventory_ip == 'public') or (cluster_vars.inventory_ip == 'private')", msg: "If inventory_ip=='public', 'assign_public_ip' must be 'yes'" }
-      when: cluster_vars.type == "gce" or cluster_vars.type == "aws"
-  tags: clusterverse_clean,clusterverse_create,clusterverse_dynamic_inventory,clusterverse_config,clusterverse_readiness
+#     - assert: { that: "(cluster_vars.assign_public_ip == 'yes' and cluster_vars.inventory_ip == 'public') or (cluster_vars.inventory_ip == 'private')", msg: "If inventory_ip=='public', 'assign_public_ip' must be 'yes'" }
+#       when: cluster_vars.type == "gce" or cluster_vars.type == "aws"
+#   tags: clusterverse_clean,clusterverse_create,clusterverse_dynamic_inventory,clusterverse_config,clusterverse_readiness

```

Comment:
Why are we commenting out here?

Reply:
That's my bed, forgotten to revert that.